packages/data-context/src/DataContext.ts

```diff
@@ -52,12 +60,15 @@ export class DataContext extends DataContextShell {
       // Fetch the browsers when the app starts, so we have some by
       // the time we're continuing.
       this.actions.app.refreshBrowsers(),
-      // load projects from cache on start
-      this.actions.project.loadProjects(),
       // load the cached user & validate the token on start
       this.actions.auth.getUser(),
     ]
 
+    if (this._config._internalOptions.loadCachedProjects) {
```

Comment:
Loading cached projects leads to test pollution for any tests concerning the projects list since the cache is empty in CI and different between different developers' machines. This allows the e2e test infrastructure to skip loading cached projects while defaulting to loading them in production. @tgriesser I'm open to other ideas on how to implement this.

Reply:
Yeah, that's fine - going to be cleaning up a lot of DataContext things in #18776 and will add this to the list of things to look at